### DISCUSSION TITLE
Label PRs that I'm an assigned reviewer of in open PR list

### DISCUSSION THREAD

**Discussion by Saare** [community/community#9974] [Pull Requests]
When opening the open PR list (`/pulls`), it would be nice to have a UI indication which PRs I'm an assigned reviewer of (without having to filter those explicitly).

**Comment by yitzchak-ben-ezra-ecoplant** [comment]
no need in a "label"
just allow us to see all our PRs (in which we are reviewers) in a single list (without filtering out those PRs which we already dropped a review)
